src/utils.js

```diff
@@ -38,6 +38,10 @@ const setupCml = async opts => {
       version !== 'latest' ? `@${version}` : ''
     }`
   );
+  console.log(
+    `Targeted CML version: ${version}, received:`,
+    await exec('npm view @dvcorg/cml version')
```

Comment:
As far as I know, [`npm view`](https://docs.npmjs.com/cli/v7/commands/npm-view) queries the package registry and prints the last version available there, not the installed version. Did you mean the following?

```bash
npm list --json --global @dvcorg/cml | jq --raw-output ".dependencies['@dvcorg/cml'].version"
```